app/subscriber/src/features/my-reports/edit/content/stories/ContentForm.tsx

```diff
@@ -152,6 +183,19 @@ export const ContentForm: React.FC<IContentFormProps> = ({
             onContentChange?.(values);
           }}
         />
+        <Formik initialValues={versions} validationSchema={sentimentFormSchema} onSubmit={() => {}}>
+          {() => (
+            <Form>
+              <FormikSentiment
```

Comment:
How is this component able to show the current value the user selected when the page is reloaded?

Reply:
Through using handleSentimentChange callback which is executed when the user clicks on the `tone` button.

Reply:
That function saves the value.  The component has no way to know the current value.  If you refresh the page it wouldn't know how to get the value.